packages/cli/api-importers/asyncapi-to-ir/src/converters/ParameterConverter.ts

```diff
@@ -81,13 +82,14 @@ export class ParameterConverter extends AbstractConverter<AsyncAPIConverterConte
                 enum: this.parameter.enum,
                 default: this.parameter.default,
                 example: this.parameter.examples?.[0],
-                examples: undefined,
+                examples: Object.values(this.parameter.examples ?? {}),
```

Comment:
<img width="776" alt="Screenshot 2025-04-05 at 12 40 35 PM" src="https://github.com/user-attachments/assets/d482ebf6-0693-47e6-a01c-f4c49fd63a25" />

do you want only the `value` field in the `examples` property?